build_run.sh

```diff
@@ -0,0 +1,10 @@
+#RUN npm install --unsafe-perm=true
```

Comment:
I noticed that you added a .gitignore that specified .sh files, did you mean to add these?